Formula/libeatmydata.rb

```diff
@@ -0,0 +1,37 @@
+class Libeatmydata < Formula
+  desc "Because fsync() should be a no-op"
+  homepage "https://www.flamingspork.com/projects/libeatmydata/"
+  url "https://github.com/stewartsmith/libeatmydata/releases/download/v130/libeatmydata-130.tar.gz"
+  sha256 "48731cd7e612ff73fd6339378fbbff38dd3bcf6c243593b0d9773ca0051541c0"
+  license "GPL-3.0-or-later"
+  head "https://github.com/stewartsmith/libeatmydata.git", branch: "master"
+
+  depends_on "autoconf"         => :build
+  depends_on "autoconf-archive" => :build
+  depends_on "automake"         => :build
+  depends_on "libtool"          => :build
+  depends_on "coreutils"        => :test
+
+  on_linux do
+    depends_on "strace" => :test
+  end
+
+  def install
+    system "autoreconf", "--force", "--install", "--verbose"
```

Comment:
I actually didn't try - I've run into problems when trying to build on ARM64 Mac's when using older configure scripts, usually regarding a checks for thread-local storage class or similar, which have been corrected in the latest autoconf, so regenerating configure is something I do without much thought to avoid potential issues.  I also don't have a M1 Mac myself to directly test on.

In this case, it shouldn't be necessary since the configure script was generated recently and with up to date tooling.

I'll try it without.

Reply:
In https://github.com/Homebrew/homebrew-core/pull/106536/commits/e0369b4319b26ad83e8c35a376b980622df85e47

Reply:
Not new enough, it seems:
```
  libeatmydata:
    * Libraries were compiled with a flat namespace.
      This can cause linker errors due to name collisions, and
      is often due to a bug in detecting the macOS version.
```
I'm fine with keeping `autoreconf`.

Reply:
Yes - seems autoconf is still needed.